src/Microsoft.Owin.Security.Authorization/AuthorizationHelper.cs

```diff
@@ -32,53 +29,25 @@ public AuthorizationHelper(IOwinContextAccessor owinContextAccessor)
                 throw new ArgumentNullException(nameof(authorizeAttribute));
             }
 
-            AuthorizationOptions options;
+            AuthorizationDependencies dependencies;
             if (controller != null)
             {
-                options = controller.AuthorizationOptions;
+                var dependenciesProvider = AuthorizationDependenciesProvider.CreateDefault();
+                dependencies = dependenciesProvider.OnCreate?.Invoke(controller.AuthorizationOptions, null);
```

Comment:
This assumes that asp.net classic programmers will always want the default `AuthorizationDependenciesProvider`.  I don't think we can make that assumption.